test/string-class-test.js

```diff
@@ -0,0 +1,169 @@
+var expect = require('chai').expect;
+require('../public/src/string-class');
+
+describe('String Class', () => {
+
+  describe('hasVowels', () => {
+    it('should return true for strings that has vowels in them', () => {
```

Comment:
### Critical

...return true for strings the `has` vowels should be 
...return true for strings the `have` vowels

please find out the reason for that and rectify accordingly.